modules/jaeger-install-elasticsearch.adoc

```diff
@@ -11,7 +11,7 @@ The default Jaeger deployment uses in-memory storage because it is designed to b
 
 .Prerequisites
 * Access to the {product-title} web console.
-* An account with the `cluster-admin` role.
+* An account with the `cluster-admin` role. If you are using {product-dedicated}, you must have an account with the `dedicated-admin` role.
```

Comment:
I'll share a sugg for here and line 28, up to you: s/If you are using/If you use/

Reply:
I agree and applied it throughout.